tests/src/FunctionalJavascript/Form/AuthenticationFormJsTest.php

```diff
@@ -270,8 +270,8 @@ protected function validateForm(callable $visitFormAsAdmin): void {
     // Test invalid password.
     $random_pass = $this->randomString();
     $page->fillField('Password', $random_pass);
-    $this->assertSendRequestMessage('.messages--error', "Failed to connect to Apigee Edge. The given username ({$this->username}) or password is incorrect. Error message: Unauthorized");
-    $web_assert->elementContains('css', 'textarea[data-drupal-selector="edit-debug-text"]', 'HTTP/1.1 401 Unauthorized');
+    $this->assertSendRequestMessage('.messages--error', "Failed to connect to Apigee Edge. Error message: Invalid host or public key. Please provide the valid host and key value.");
```

Comment:
This new error message by the MGMT API does not make any sense :)

Reply:
It looks like a bug in the management API to me but I think we should open a ticket and handle this in a new issue.

Reply:
Indeed, this is unrelated, but because of this change in the MGMT API, all test fail in all builds.
This problem rather made me thinking, now I think we should not expose the original/raw error message to end-users just like this. We implemented in this way because we thought that it is the best for end-users and these particular messages will not change and the MGMT API should be able to tell the best what went wrong.
Although, after this problem occurred, maybe we should not display the MGMT API error messages we should display a translation of them. We could still display the original MGMT API error message in the Debug window.